### ISSUE TITLE
Low Training Accuracy

### ISSUE THREAD

**Issue by mayank0403** [pps-lab/fl-analysis#1]
Hi,

I am getting very low accuracy when running training without any malicious clients. For ResNet18 for Cifar10, the accuracy peaks at about 0.25 and starts getting worse after that with increasing rounds.

I am using the same parameters as mentioned in the RoFL paper, here is my `config.yml` file (accuracy stays bad even when I remove clipping):
```
environment:
  num_clients: 100
  num_selected_clients: 40
  num_malicious_clients: 0
  experiment_name: "Sample run without attackers"

load_model: ./models/resnet18.h5

server:
  num_rounds: 20
  num_test_batches: 50
  aggregator:
    name: FedAvg
  global_learning_rate: 1

client:
  clip:
    type: l8
    value: 10
  model_name: resnet18
  benign_training:
    num_epochs: 2
    batch_size: 64
    optimizer: SGD
    learning_rate: 0.02
    step_decay: true

dataset:
  dataset: cifar10
  data_distribution: nonIID

```

Since the readme on the repo says that the included models are pre-trained, do you know why am I seeing such low accuracy? Here is a sample run for 2 rounds:
```
INFO:root:Starting training...
round= 0 	test_accuracy= 0.0859375 	adv_success= 0 	test_loss= 4.521559
DEBUG:root:Memory info: 2522755072
round= 1 	test_accuracy= 0.10375 	adv_success= 0 	test_loss= 2.4279819 	duration= 270.714478969574
DEBUG:root:Memory info: 2390228992
```

For a pre-trained model, shouldn't the `test_accuracy` of round 0 be already good?

Thanks for the help in advance!

**Comment by mayank0403** [comment]
There was an indentation issue with the config file.

**Comment by hiddely** [comment]
Hi, completely missed that you created an issue until now, my bad!

Glad you found the problem already!
